src/shared/ui/InputFiled/InputTypeFiled.tsx

```
@@ -0,0 +1,59 @@
+
+  return (
+    <div className="flex flex-col gap-1">
+      <div
+        className={cn(
+          S.defaultInputArea,
+          (isFocused || showSuccess) && 'border-secondary',
+          isBlur && 'border-gray-700',
+          showError && 'border-dangerous',
+          className
+        )}
+        tabIndex={0}
```

Biome (2.1.2)

[error] The HTML element div is non-interactive. Do not use tabIndex.

Adding non-interactive elements to the keyboard navigation flow can confuse users.
Unsafe fix: Remove the tabIndex attribute.


(lint/a11y/noNoninteractiveTabindex)